tests/test_deviation_survey.py

```diff
@@ -0,0 +1,141 @@
+import os
+import numpy as np
+from numpy.testing import assert_array_almost_equal
+import pandas as pd
+import pytest
+
+from resqpy.model import Model
+import resqpy.well
+
+
+def test_DeviationSurvey(example_model_with_well, tmp_path):
+    # Test that all attributes are correctly saved and loaded from disk
+
+    # --------- Arrange ----------
+    # Create a Deviation Survey object in memory
+
+    # Load example model from a fixture
+    model, well_interp, datum, traj = example_model_with_well
+    epc_path = model.epc_file
+
+    # Create 3 copies of the survey, using different initialisers
+    data = dict(
+        title = 'Majestic Umlaut ö',
+        originator = 'Thor, god of sparkles',
+        md_uom = 'ft',
+        angle_uom = 'rad',
```

Comment:
The inclinations need to be between +/- 180 degrees, so between +/- π radians.

Azimuths need to be between 0 and 360 degrees, or 0 and 2π radians (which the current data is)